src/main/java/mate/jdbc/service/impl/CarServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id);
```

Comment:
```suggestion
        return carDao.get(id).orElseThrow(
                () -> new NoSuchElementException("Can't get car by this id: " + id));
```
cause u return optional from dao, so need to throw exception in case if there no car